src/test/java/library/view/function/BookFunctionTest.java

```diff
@@ -0,0 +1,22 @@
+package library.view.function;
+
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+import java.util.Optional;
+
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+class BookFunctionTest {
+
+    @Test
+    @DisplayName("도서 기능 코드로 도서 기능을 찾을 수 있어야 합니다.")
+    void testFindByCode() {
+        Optional<BookFunction> foundBookFunction = BookFunction.findByCode("1");
+        assertTrue(foundBookFunction.isPresent());
+
+        Optional<BookFunction> notFoundBookFunction = BookFunction.findByCode("8");
+        assertFalse(notFoundBookFunction.isPresent());
+    }
```

Comment:
```suggestion
    @ValueSource(strings = {"1", "2"})
    @ParameterizedTest(name = "도서 기능 코드가 {0}인 경우 도서 기능을 찾을 수 있어야 합니다.")
    void testFindByCode(String code) {
        Optional<BookFunction> foundBookFunction = BookFunction.findByCode(code);
        assertTrue(foundBookFunction.isPresent());
    }
```

code 값이 문자열이고, 둘 이상의 값을 검증하는 방법으로 @ValueSource를 사용하는 방법을 권장합니다.
만약에 code 값이 Enum으로 정의되어 있다면 @EnumSource도 좋습니다.

Reply:
[3f3ad25](https://github.com/prgrms-be-devcourse/java-library-management/pull/21/commits/3f3ad25adad4580486141a7c06819acd6cd9856a)
반영했습니다!